src/basho_bench_driver_cluster.erl

```diff
@@ -0,0 +1,59 @@
+%% -------------------------------------------------------------------
+%%
+%% basho_bench: benchmark service on any within Erlang cluster 
+%%              using distribution protocol
+%%
+%% Copyright (c) 2015 Dmitry Kolesnikov
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+-module(basho_bench_driver_cluster).
+
+-export([
+   new/1,
+   run/4
+]).
+
+-record(state, {actor}).
+
+%% ====================================================================
+%% API
+%% ====================================================================
+
+new(Id) ->
+    Actors = basho_bench_config:get(cluster_actors, []),
+    {Name, Node} = Actor = lists:nth(random:uniform(length(Actors)), Actors),
```

Comment:
You are absolutely right! Round-robin allocation is cleaner and gives better way to control number of workers vs number of "remote" processes. 

There was not any reason why random selection was done in original version. I've made changes based on your suggestion.